### PR TITLE
Adding copayer name

### PR DESCRIPTION
**Pull request by matiaspando**

### CODE REVIEW

File: public/views/join.html [hunk 1]
```diff
@@ -2,6 +2,25 @@
   <div class="row m20t">
     <div class="large-12 columns">
       <form name="joinForm" ng-submit="join.join(joinForm)" novalidate>
+            <div class="row collapse">
+            <label for="name" class="left"><span translate>Your name</span> 
+              <small translate ng-show="joinForm.name.$pristine">Required</small>
+            </label>
+            <span class="has-error right size-12" ng-show="joinForm.name.$invalid
```
Comment (matiu) [inline public/views/join.html]: please remove this. It should be the same input as createWallet.

**Comment by matiu** [comment]
also, need rebase.

**Comment by matiaspando** [comment]
Updated!

**Comment by matiaspando** [comment]
Closes #2490

**Pull request merged**